scripts/ligandtocore/QD.py

```diff
@@ -0,0 +1,204 @@
+import copy
+import itertools
+import time
+import sys
+
+from scm.plams import (Atom, MoleculeError, Settings)
+from qmflows import molkit
```

Comment:
@BvB93 change this import to **import scm.plams.interfaces.molecule.rdkit as molkit**